IS7024Project/Pages/Index.cshtml

```diff
@@ -91,7 +92,15 @@
         </div>
     </div>
     <table border="1" width="90%">
-        <tr><th>Incident #</th><th>Offense</th><th>Location</th><th>Date</th><th>Day of Week</th><th>CC Neighborhood</th><th>CPD Neighborhood</th><th>Latitude</th><th>Longitude</th><th>Zip</th></tr>
+        <tr>
```

Comment:
I organized table code for better visualization in code editor.

Reply:
A bit easier to read, yes.